scene/resources/dynamic_font.cpp

```diff
@@ -79,13 +79,20 @@ void DynamicFontData::set_force_autohinter(bool p_force) {
 
 void DynamicFontData::set_use_antialias(bool p_use) {
 	use_antialias = p_use;
-}	
+}
+
+bool DynamicFontData::get_use_antialias() const {
+	return use_antialias;
+}
 
 void DynamicFontData::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_font_path", "path"), &DynamicFontData::set_font_path);
 	ClassDB::bind_method(D_METHOD("get_font_path"), &DynamicFontData::get_font_path);
+	ClassDB::bind_method(D_METHOD("set_use_antialias", "antialias"), &DynamicFontData::set_use_antialias);
```

Comment:
The more idiomatic bindings would be: `"set_use_antialias", "enabled"` and `"is_using_antialias"`.